lib/Jsrt/ChakraCore.h

```diff
@@ -42,66 +48,115 @@ typedef void* JsModuleRecord;
 /// </remarks>
 typedef void *JsSharedArrayBufferContentHandle;
 
+/// <summary>
+///    Flags for parsing a module.
+/// </summary>
 typedef enum JsParseModuleSourceFlags
 {
+    /// <summary>
+    /// Module source is UTF16.
+    /// </summary>
     JsParseModuleSourceFlags_DataIsUTF16LE = 0x00000000,
+    /// <summary>
+    /// Module source is UTF8.
+    /// </summary>
     JsParseModuleSourceFlags_DataIsUTF8 = 0x00000001
 } JsParseModuleSourceFlags;
 
+/// <summary>
+///    The types of host info that can be set on a module record with JsSetModuleHostInfo.
+/// </summary>
+/// <remarks>
+/// For more information see JsSetModuleHostInfo.
+/// </remarks>
 typedef enum JsModuleHostInfoKind
 {
+    /// <summary>
+    /// An exception object - e.g. if the module file cannot be found.
+    /// </summary>
     JsModuleHostInfo_Exception = 0x01,
+    /// <summary>
+    /// Host defined info.
+    /// </summary>
     JsModuleHostInfo_HostDefined = 0x02,
+    /// <summary>
+    /// Callback for receiving notification when module is ready.
+    /// </summary>
     JsModuleHostInfo_NotifyModuleReadyCallback = 0x3,
+    /// <summary>
+    /// Callback for receiving notification to fetch a dependent module.
+    /// </summary>
     JsModuleHostInfo_FetchImportedModuleCallback = 0x4,
+    /// <summary>
+    /// Callback for receiving notification for calls to ```import()```
```

Comment:
Only for `import()` in "legacy" scripts - `import()` in module code uses the standard FIM callback.